common-graphql/src/main/scala/queries/common/GeneratedSequenceSQL.scala

```diff
@@ -14,6 +14,18 @@ import lucuma.schemas.decoders.given
 import lucuma.schemas.odb.*
 
 object GeneratedSequenceSQL {
+  given offsetComponentDecoder[T]: Decoder[Offset.Component[T]] = Decoder.instance(c =>
```

Comment:
There's a decoder already in `lucuma-schemas`.

Reply:
https://github.com/gemini-hlsw/lucuma-schemas/blob/main/model/src/main/scala/lucuma/schemas/decoders/CoreModelDecoders.scala#L143

Reply:
Hmm, I just moved this around. I guess we should remove it